engine-rest/engine-rest/src/main/java/org/camunda/bpm/engine/rest/dto/task/TaskDto.java

```diff
@@ -54,6 +55,44 @@ public class TaskDto {
    */
   private String taskState;
 
+  public TaskDto() {
+  }
+
+  public TaskDto(Task task) {
+    this.id = task.getId();
+    this.name = task.getName();
+    this.assignee = task.getAssignee();
+    this.created = task.getCreateTime();
+    this.lastUpdated = task.getLastUpdated();
+    this.due = task.getDueDate();
+    this.followUp = task.getFollowUpDate();
+
+    if (task.getDelegationState() != null) {
+      this.delegationState = task.getDelegationState().toString();
+    }
+
+    this.description = task.getDescription();
+    this.executionId = task.getExecutionId();
+    this.owner = task.getOwner();
+    this.parentTaskId = task.getParentTaskId();
+    this.priority = task.getPriority();
+    this.processDefinitionId = task.getProcessDefinitionId();
+    this.processInstanceId = task.getProcessInstanceId();
+    this.taskDefinitionKey = task.getTaskDefinitionKey();
+    this.caseDefinitionId = task.getCaseDefinitionId();
+    this.caseExecutionId = task.getCaseExecutionId();
+    this.caseInstanceId = task.getCaseInstanceId();
+    this.suspended = task.isSuspended();
+    this.tenantId = task.getTenantId();
+    this.taskState = task.getTaskState();
+    try {
+      this.formKey = task.getFormKey();
+      this.camundaFormRef = task.getCamundaFormRef();
+    }
+    catch (BadUserRequestException e) {
+      // ignore (initializeFormKeys was not called)
```

Comment:
Note to myself: 
❓ Is this problematic? Can some other issue be swallowed by this?

Reply:
We already do this in `#fromEntity()`, so all good.